eng/config/PublishData.json

```diff
@@ -45,13 +45,13 @@
       "vsBranch": "rel/d16.3",
       "vsMajorVersion": 16
     },
-    "master-vs-deps": {
+    "release/dev16.3-vs-deps": {
       "nugetKind": ["Shipping", "NonShipping"],
       "version": "3.3.*",
       "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
       "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-      "channels": [ "dev16.3", "dev16.3p3" ],
-      "vsBranch": "master",
+      "channels": [ "dev16.3" ],
+      "vsBranch": "lab/d16.3stg",
```

Comment:
There is no lab/d16.3stg branch. I think this should be master until they branch off rel/d16.3.

Reply:
Ah, thanks